### PR TITLE
Add a failing test for #65.

### PR DESCRIPTION
**Pull request by jamestalmage**
See #65.

I haven't figured out the exact cause of the problem, but a fix for #65 should make this fixture pass.

I've deployed to @sindresorhus's repo, so anyone should be able to just `checkout add-ignores-test-fixture` if they want to take a crack at solving the problem.

### CODE REVIEW

**Comment by jamestalmage** [comment]
5010bd7 is probably worthwhile on it's own.

**Comment by sindresorhus** [comment]
:+1:

File: test/fixtures/ignores/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "application-name",
+  "version": "0.0.1",
+  "xo": {
+    "ignore": [
+      "{test/,}fixture{s,}/**"
```
Comment (sindresorhus) [inline test/fixtures/ignores/package.json]: @jamestalmage Looking at this again, should it be prefixed with `!`?

Comment (jamestalmage) [inline test/fixtures/ignores/package.json]: yes.

**Pull request merged**